src/main/java/model/RacingGame.java

```diff
@@ -1,69 +1,26 @@
 package model;
 
-import view.ResultView;
-
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
 
 public class RacingGame {
-    private final List<Car> carList;
-
-    public RacingGame(int numOfCars) {
-        this(numOfCars, new RandomMove());
-    }
-
-    public RacingGame(int numOfCars, MoveStrategy moveStrategy) {
-        this.carList = participate(numOfCars, moveStrategy);
-    }
+    private final Cars cars;
 
     public RacingGame(List<Car> cars) {
-        this.carList = cars;
-    }
-
-    private List<Car> participate(int numOfCars, MoveStrategy moveStrategy) {
-        List<Car> result = new ArrayList<>();
-        for (int i = 1; i <= numOfCars; i++) {
-            result.add(new Car(moveStrategy));
-        }
-
-        return result;
+        this.cars = new Cars(cars);
     }
 
-    public ResultView run(int turn) {
-        ResultView resultView = new ResultView();
+    public RacingRecord run(int turn) {
+        RacingRecord racingRecord = new RacingRecord(turn);
         for (int t = 1; t <= turn; t++) {
```

Comment:
```suggestion
        for (int i = 1; i <= turn; i++) {
```
단순 반복용 for문이니까 t 보다는 그냥 관례상 i를 써도 괜찮을듯합니다.
turn이라 t를 사용하셨나요? 😂